src/main/java/org/killbill/billing/plugin/adyen/client/AdyenConfigProperties.java

```diff
@@ -200,6 +204,17 @@ public AdyenConfigProperties(final Properties properties, final String currentRe
         }
     }
 
+    private Period readPendingHppPaymentWithouCompletionExpirationPeriod(final Properties properties) {
+        final String value = properties.getProperty(PROPERTY_PREFIX + "pendingHppPaymentWithoutCompletionExpirationPeriod");
+        if (value != null) {
+            try {
+                return Period.parse(value);
+            } catch (IllegalArgumentException e) { /* Ignore */ }
```

Comment:
Nit: `final` (note that if you import the per project settings from `.idea/`, this should be taken care of for you).